tests/test_account.py

```diff
@@ -47,7 +47,7 @@ def test_cannot_login_passwordless_if_email_in_invalid_format(self, base_url, se
         invalid_email = "invalidmail"
         auth0.enter_email(invalid_email)
         auth0.click_send_email()
-        error_login_confirmation_message = 'We were unable to send the email : error in email - email format validation failed: {0}'\
+        error_login_confirmation_message = 'We were unable to send the email. Please ensure you submitted a correctly formatted e-mail address.'\
```

Comment:
Updated, as per changes in https://github.com/mozilla-iam/auth0-deploy/pull/104/files